admin/controller/payment/razorpay.php

```diff
@@ -311,6 +311,359 @@ public function uninstall(): void {
+        }
+
+        if (isset($this->request->get['filter_plan_name']))
+        {
+            $url .= '&filter_plan_name=' . urlencode(html_entity_decode($this->request->get['filter_plan_name'], ENT_QUOTES, 'UTF-8'));
+        }
+
+        if (isset($this->request->get['filter_plan_status']))
+        {
+            $url .= '&filter_plan_status=' . $this->request->get['filter_plan_status'];
+        }
+
+        if (isset($this->request->get['filter_date_created']))
+        {
+            $url .= '&filter_date_created=' . $this->request->get['filter_date_created'];
+        }
+
+        if (isset($this->request->get['sort']))
+        {
+            $url .= '&sort=' . $this->request->get['sort'];
+        }
+
+        if (isset($this->request->get['plan']))
+        {
+            $url .= '&plan=' . $this->request->get['plan'];
+        }
+
+        if (isset($this->request->get['page']))
+        {
+            $url .= '&page=' . $this->request->get['page'];
+        }
+
+        $data['breadcrumbs'] = array();
+
+        $data['breadcrumbs'][] = array(
+            'text' => $this->language->get('text_home'),
+            'href' => $this->url->link('common/dashboard', 'user_token=' . $this->session->data['user_token'], true)
+        );
+
+        $data['breadcrumbs'][] = array(
+            'text' => $this->language->get('plan_title'),
+            'href' => $this->url->link('extension/razorpay/payment/razorpay.getPlan', 'user_token=' . $this->session->data['user_token'] . $url, true)
+        );
+
+        $data['plans'] = array();
+
+        $filter_data = array(
+            'filter_plan_id'        => $filter_plan_id,
+            'filter_plan_name'         => $filter_plan_name,
+            'filter_plan_status'    => $filter_plan_status,
+            'filter_date_created'      => $filter_date_created,
+            'sort'                   => $sort,
+            'order'                  => $order,
+            'start'                  => ($page - 1) *10,
+            'limit'                  =>10
+        );
+        $plan_total = $this->model_extension_razorpay_payment_razorpay->getTotalPlan($filter_data);
+
+        $results = $this->model_extension_razorpay_payment_razorpay->getPlans($filter_data);
+
+        foreach ($results as $result)
+        {
+            $data['plans'][] = array(
+                'entity_id'      => $result['entity_id'],
+                'plan_id'      => $result['plan_id'],
+                'plan_name'    => $result['plan_name'],
+                'plan_desc'     => $result['plan_desc'],
+                'name'          => $result['name'],
+                'plan_type'     => $result['plan_type'],
+                'plan_frequency'     => $result['plan_frequency'],
+                'plan_bill_cycle'     => $result['plan_bill_cycle'],
+                'plan_trial'     => $result['plan_trial'],
+                'plan_bill_amount'     => $result['plan_bill_amount'],
```

Comment:
@abdulwahidsharief , please format it so that all `=>` will start at same place in each line.